golang/testsuite/testsuite_impl/files_artifact_mounting_test/files_artifact_mounting_test_.go

```diff
@@ -47,34 +50,31 @@ func (f FilesArtifactMountingTest) Configure(builder *testsuite.TestConfiguratio
 }
 
 func (f FilesArtifactMountingTest) Setup(networkCtx *networks.NetworkContext) (networks.Network, error) {
-	configFactory := nginx_static.NewNginxStaticContainerConfigFactory(testFilesArtifactId)
-	_, hostPortBindings, availabilityChecker, err := networkCtx.AddService(fileServerServiceId, configFactory)
-	if err != nil {
-		return nil, stacktrace.Propagate(err, "An error occurred adding the file server service")
-	}
-	if err := availabilityChecker.WaitForStartup(waitForStartupTimeBetweenPolls, waitForStartupMaxRetries); err != nil {
-		return nil, stacktrace.Propagate(err, "An error occurred waiting for the file server service to start")
-	}
-	logrus.Infof("Added file server service with host port bindings: %+v", hostPortBindings)
 	return networkCtx, nil
 }
 
 func (f FilesArtifactMountingTest) Run(network networks.Network) error {
-	// Only necessary because Go doesn't have generics
-	castedNetwork := network.(*networks.NetworkContext)
+	// Necessary because Go doesn't have generics
+	castedNetworkContext := network.(*networks.NetworkContext)
 
-	uncastedService, err := castedNetwork.GetService(fileServerServiceId)
+	configFactory := nginx_static.NewNginxStaticContainerConfigFactory(testFilesArtifactId)
```

Comment:
Same question - why are we moving setup stuff into the run method?

Reply:
I'll return the addService to Setup method